aws/src/database/multi_table/update_cautionary_alerts.py

```diff
@@ -0,0 +1,60 @@
+
+from mypy_boto3_dynamodb.service_resource import Table
+
+from aws.src.database.dynamodb.utils.get_dynamodb_table import get_dynamodb_table
+from aws.src.utils.csv_to_dict_list import csv_to_dict_list
+from enums.enums import Stage
+from dateutil import parser
+from datetime import datetime
+
+STAGE = Stage.HOUSING_PRODUCTION
+
+def update_cautionary_alert_in_db_if_tenure_inactive(person_table: Table, alerts_from_csv: list[dict]):
+    """
+    Iterate through the alerts and remove property details if tenure is inactive
+    1. Get person details from dynamo using mmh_id in csv
+    2. check if there is an end date within the tenure object
+    3. if so remove the property details from the csv
+    """
+
+    for alert_item in alerts_from_csv:
+        alert_id = alert_item["id"]
+        mmh_id = alert_item["mmh_id"]
+        if mmh_id is '' or None:
+            alert_item["failed_reason"] = f"mmh_id is null for id {alert_id}"
+            continue
+        person = person_table.get_item(Key={"id": mmh_id}).get("Item")
+
+        person_id = person["id"]
+        if person is None:
+            alert_item["failed_reason"] = f"Could not get person for personId {person_id}"
+            continue
+        tenures = person["tenures"]
```

Comment:
Just to confirm... When the Person object within the DynamoDB has no tenures under it, does its "tenures" field default to empty array, or None?

Reply:
Ah okay, because the question was... would anything unexpected happen if we had:
``` py
for tenure_in_person in None:
```
?